### PR TITLE
11266 - Secondary Footer Contents and Styles

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP11266](https://maps.tpondemand.com/entity/11266-secondary-footer-panels-content)

This PR aims to implement the contents on the existing structure for the two bottom footer panels and its inherent styles and responsive behaviour.

A new piece of work has resulted from this PR which should address the way accessibility problems are submitted via the footer button, a placeholder ticket was created at [TP11409](https://maps.tpondemand.com/entity/11409-footer-accessibility-button). Currently, this button is not working.

Translations for the Footer were not in the scope of this PR and they should be implemented in [TP11281](https://maps.tpondemand.com/entity/11281-footer-contact-panels-tbd).

Additionally, the Button component was refactored to accommodate a new prop for the blog styles and the way the styles for the multiple states are applied were also modified.

### CODE REVIEW

File: src/components/Footer/FooterPrimary/StyledFooterPrimary.js [hunk 1]
```diff
@@ -1,18 +1,78 @@
...
   ${props =>
     !props.background && `background-color: ${props.theme.colors.alternate};`}
-  ${props => !props.padding && `padding: 24px 0 12px;`}
+  ${props => !props.padding && `padding: 24px 15px 12px;`}
 `
 
-const SocialMediaLinks = styled(Col)``
+// social media section
+const SocialContainer = styled(Col)`
+  ${props => !props.padding && `padding: 12px 0;`}
```
Comment (davidtrussler) [inline src/components/Footer/FooterPrimary/StyledFooterPrimary.js:19]: I think I would make the value of `justify-content` to be `space-around` here to give the social media icons a bit more breathing space.

File: src/components/Footer/FooterPrimary/StyledFooterPrimary.js [hunk 2]
```diff
@@ -1,18 +1,78 @@
...
+
+const SocialLogo = styled(Anchor)`
+  ${props => props.facebook && `background-color: #3c5a99;`}
+  ${props => props.twitter && `background-color: #2ba9e1;`}
+  ${props => props.youtube && `background-color: #cd201f;`}
+
+  border-radius: 50%;
+  width: 60px; 
+  height: 60px;
+  line-height: 57px;
```
Comment (davidtrussler) [inline src/components/Footer/FooterPrimary/StyledFooterPrimary.js:40]: Pretty trivial I know but maybe `line-height` should be the same value as the height.

File: src/components/Button/buttonStyles.js [hunk 3]
```diff
@@ -17,9 +17,38 @@ const defaultStyle = props => css`
...
+  &:hover,
+  &:active {
+    ${props =>
+      !props.background &&
+      `background-color: ${props.theme.colors.button.activeBackground};`}
+    ${props =>
+      !props.border &&
+      `border-color: ${props.theme.colors.button.activeBorderColor};`}
+  }
+
```
Comment (davidtrussler) [inline src/components/Button/buttonStyles.js:33]: I'm not sure exactly where these are set but it looks like the focus styles are not quite right: we are setting the outline correctly but the colour of the font should not change and the background colour should be the same as the outline.

Comment (andreferreiradlw) [reply src/components/Button/buttonStyles.js:33]: not sure if I understand entirely but the font color doesn't change when focused and you're right, there's an outline missing when focused

Comment (davidtrussler) [reply src/components/Button/buttonStyles.js:33]: I'm looking at this for instance - seems like the outline's fine but the text has changed colour and there's no background. 
image

Comment (andreferreiradlw) [reply src/components/Button/buttonStyles.js:33]: That seems to be the anchor and not a button, let me investigate

**Pull request merged**